mostlyai/qa/_filesystem.py

```diff
@@ -266,14 +285,13 @@ def biv_multi_index(bin, col1, col2):
         return trn_cnts_uni, trn_cnts_biv
 
     def store_pca_model(self, pca_model: PCA):
-        with self.pca_model_path.open("wb") as file:
-            pickle.dump(pca_model, file)
+        sio.dump(pca_model, self.pca_model_path)
 
     def load_pca_model(self) -> PCA | None:
-        if not self.pca_model_path.exists():
-            return None
-        with self.pca_model_path.open("rb") as file:
-            return pickle.load(file)
+        unknown_types = sio.get_untrusted_types(file=self.pca_model_path)
```

Comment:
What would happen when `not self.pca_model_path.exists()`? Probably we'd want to gracefully log and skip - same behaviour as before the change.